compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp

```diff
@@ -980,6 +980,24 @@ void quantize_const_inputs(luci::CircleNode *node, loco::DataType output_type)
         quant_const(const_node, output_type);
       break;
 
+    case luci::CircleOpcode::PADV2:
+      // Only the third input is quantized
```

Comment:
First input can be quantized if it is a constant tensor.

Reply:
Updated this commentary.